modular_citadel/code/modules/reagents/reagents/cit_reagents.dm

```diff
@@ -164,6 +164,25 @@
 		M.adjustArousalLoss(2)
 	..()
 
+/datum/reagent/drug/pheromones
+	name = "slime pheromones"
+	id = "pheromones"
+	description = "Pheromones released by slimes, in liquid concentrated form. Has about the same strength of Hexacrocin, but without the brain damage and addiction."
+	taste_description = "ants"
+	color = "#FFADFF"
+
+/datum/reagent/drug/pheromones/on_mob_life(mob/living/M)
```

Comment:
Copypasta minus moaning and trait immunity check, make this a subtype of aphrodisiacplus with the addiction and overdose values set to 0. If you want to retain the no emoting and immunity bypass, just make a couple additional vars for them.